crypto/rsa/rsa_sp800_56b_gen.c

```diff
@@ -353,6 +374,7 @@ int rsa_sp800_56b_pairwise_test(RSA *rsa, BN_CTX *ctx)
     k = BN_CTX_get(ctx);
     if (k == NULL)
         goto err;
+    BN_set_flags(k, BN_FLG_CONSTTIME);
```

Comment:
What about `tmp` here?

Reply:
I quickly checked and it seems that functions using `tmp` do not check any `BN_FLG_CONSTTIME`, so I believe is ok.

Reply:
Hmm....`tmp` is used in the call to `BN_mod_exp` below, which then calls `BN_mod_exp_mont`, which then calls `BN_mod_exp_mont_consttime` if the flag is set...or am I misreading the logic somewhere? In practice `rsa->d` should already have the flag set, so it should still go down that path...but still....

Reply:
I actually originally flagged tmp and later removed it:

From a logic perspective k and rsa->d here are the secrets, tmp is the public encrypted value and e and n are public.

Flagging tmp here could mask more serious mistakes from being detected by the CI, that was my rationale.

Reply:
Ok - makes sense.